internal/clients/team4/client.go

```diff
@@ -46,22 +44,37 @@ func newClientInternal(clientID shared.ClientID, testing *testing.T) client {
 		iitoObs: &iitoObs,
 	}
 
-	judgeHistory := map[uint]map[shared.ClientID]judgeHistoryInfo{}
+	judgeHistory := accountabilityHistory{
+		history: map[uint]map[shared.ClientID]judgeHistoryInfo{},
+		updated: false,
+	}
 
 	emptyRuleCache := map[string]rules.RuleMatrix{}
+	emptyTrust := trust{
+		trustMap: map[shared.ClientID]float64{},
+	}
+
+	importancesMatrix := importances{
+		requestAllocationImportance:                mat.NewVecDense(6, []float64{5.0, 1.0, -1.0, -1.0, 5.0, 1.0}),
+		commonPoolResourceRequestImportance:        mat.NewVecDense(6, []float64{4.0, 1.0, -1.0, -1.0, 1.0, 1.0}),
+		resourceReportImportance:                   mat.NewVecDense(6, []float64{5.0, 5.0, -5.0, -5.0, 1.0, 5.0}),
+		getTaxContributionImportance:               mat.NewVecDense(4, []float64{-2.0, -2.0, 4.0, 1.0}),
+		decideIIGOMonitoringAnnouncementImportance: mat.NewVecDense(3, []float64{1.0, -1.0, 1.0}),
```

Comment:
Could you add some comments to this? It'd be nice to have some reasoning behind the values

Reply:
that's to @Fabio752

Reply:
Ah, okay. I couldn't see where the values were deleted from. Looked like a section you'd written